src/information_theory/hamming.lean

```diff
@@ -174,13 +179,19 @@ lemma hamming_norm_smul [has_zero α] [Π i, smul_with_zero α (β i)] {k : α}
   (hk : ∀ i, is_smul_regular (β i) k) (x : Π i, β i) : hamming_norm (k • x) = hamming_norm x :=
 hamming_norm_comp (λ i (c : β i), k • c) hk (λ i, by simp_rw smul_zero')
 
+/-- Corresponds to `dist_zero_right`. -/
+@[simp] lemma hamming_dist_zero_right (x : Π i, β i) : hamming_dist x 0 = hamming_norm x := rfl
+
+/-- Corresponds to `dist_zero_left`. -/
+@[simp] lemma hamming_dist_zero_left : hamming_dist (0 : Π i, β i) = hamming_norm :=
+funext $ λ x, by rw [hamming_dist_comm, hamming_dist_zero_right]
+
```

Comment:
I would move these lemmas up to immediately after defining `hamming_norm`, especially since the first is `rfl`

Reply:
Which is to say: I would not have moved these in the first place.